.travis.yml

```diff
@@ -4,3 +4,15 @@ go: "1.11"
 install: true
 script:
 - GINKGO_EXTRA_ARGS="--compilers=2" make all test/unit
+
+deploy:
+- provider: script
+  script: docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io && make handler push-handler
+  skip_cleanup: true
+  on:
+    branch: master
+- provider: script
+  script: docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io && make handler push-handler HANDLER_IMAGE_SUFFIX=":"$TRAVIS_TAG
```

Comment:
@qinqon This will work?

Reply:
Do we need it with "latest is default"?

Reply:
You can just use "make push-handler" since "handler" is a dependency from "push-handler".
We don't need double quotes at any of the variables expansion (Maybe travis for us too)